topi2_1/src/test/java/com/bootcamp/app/topi2_1/AppTest.java

```diff
@@ -0,0 +1,34 @@
+package com.bootcamp.app.topi2_1;
```

Comment:
Why did you added this test class? You are not really testing the App class or am I wrong?